kotlin/src/main/com/looker/rtl/Constants.kt

```diff
@@ -119,3 +114,29 @@ internal fun Date(utcDateTime: String): Date {
     utcFormat.timeZone = TimeZone.getTimeZone("UTC")
     return utcFormat.parse(utcDateTime)
 }
+
+/** Structure read from an .INI file */
+typealias ApiSections = Map<String, Map<String, String>>
+
+/**
+ * Parse and cleanup something that looks like an .INI file, stripping outermost quotes for values
+ */
+fun apiConfig(contents: String): ApiSections {
```

Comment:
Personally, I would have kept this with the rest of the config parsing code in ApiSettings, but I'll leave that up to you.

Reply:
Because it wasn't specific to the SDK, I was leaving it in the RTL. However, it's no big deal if someone wanted to create a customized ConfigProvider with this code being in the SDK rather than the RTL, so I'll move it so it's "closer."